src/content/plugins/split-chunks-plugin.md

```diff
@@ -0,0 +1,230 @@
+---
+title: SplitChunksPlugin
+contributors:
+  - sokra
+  - jeremenichelli
+related:
+  - title: "webpack 4: Code Splitting, chunk graph and the splitChunks optimization"
+    url: https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
+---
+
+Originally in webpack, chunks (and modules imported inside them) were connected by a parent-child relationship in the internal webpack graph.
+
+This connection didn't allow to further optimizations and resulted into more downloaded code.
+
+webpack v4 removes the `CommonsChunkPlugin` in favor of `optimization.splitChunks` and `optimize.runtimeChunk` options. Here is how the new flow works.
+
+## Defaults
+
+Out of the box `SplitChunksPlugin` should work great for most users.
+
+By default it only affects on-demand chunks because changing initial chunks would affect the script tags the HTML file should include to run the project.
```

Comment:
Maybe explain why changing those initial script tags is a problem?

Reply:
Honestly, I think that the sentence is pretty clear by itself.